CHANGELOG.md

```diff
@@ -20,12 +21,12 @@
 
 ##### Enhancements
 
-* Expose the underlying `Node`'s `Mark` on `Decoder`.  
```

Comment:
Sorry @tinder-maxwellelliott one more thing, can you please revert these whitespace changes in this file? The two trailing spaces are intentional: https://github.com/jpsim/Yams/blob/main/CONTRIBUTING.md

> 2 trailing spaces on the last line describing the change.

Markdown forces a line break when it encounters a trailing double space. https://www.markdownguide.org/basic-syntax/#line-breaks